corehq/apps/analytics/tasks.py

```diff
@@ -288,6 +291,10 @@ def track_periodic_data():
                 {
                     'property': 'max_mobile_workers_in_a_domain',
                     'value': max_workers
+                },
+                {
+                    'property': 'project_spaces_created_by_user',
```

Comment:
Is the property `project_spaces_created_by_user ` created in Hubspot [portal](https://github.com/dimagi/commcare-hq/tree/created-domain-analytics/corehq/apps/analytics#addingremoving-hubspot-properties) as well?